.circleci/config.yml

```diff
@@ -191,6 +201,12 @@ workflows:
           requires:
             - test-sanity-check
 # build
+      - build-container:
```

Comment:
Why we need devenv here? It only for local development.

Reply:
>Why we need devenv here? 
I've added this for check that `devenv` builds fine to avoid next regerssions

Are you have any idea how we can check this docker file without building on CI?

Reply:
I suppose we do not need to check it on CI, since this image are only used in local builds. I suppose we could excluded it from make k8s-save builds also.